pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -548,7 +548,7 @@ spec:
       runAfter:
         - check-data-keys
         - embargo-check
-        - publish-pyxis-repository
+        - push-rpm-data-to-pyxis
```

Comment:
Can you add run-file-updates and rh-sign-image so it is truly last (before the finally and update-cr-status)? I think that is all the additional ones needed

Reply:
OK, what about rh-sign-image-cosign? It's not really used yet, but probably makes sense? Hopefully when that one's skipped, it won't block this.

Reply:
I added `rh-sign-image-cosign` as well. Let me know if you disagree.

Reply:
> OK, what about rh-sign-image-cosign? It's not really used yet, but probably makes sense? Hopefully when that one's skipped, it won't block this.

I don't know, but that should certainly be tested before we merge this

Reply:
The e2e test does create an advisory and it definitely does not run rh-sign-image-cosign, so it should cover this, no?

Reply:
No idea if the rh-sign-image-cosign task is run in e2e. My personal faith in e2e is kind of low lately as well. But you may be right. It would also be easy to test in a temp unit test file. I just don't want to merge it until we know for sure that task A having runAfter a skipped task still executes task A

Reply:
Scott will test this today.

Reply:
work of art:

![adv-perfect](https://github.com/user-attachments/assets/b243fb9c-85f5-4e8c-a868-a6eef6aaec96)